doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -985,15 +984,27 @@ Generation of induction principles with ``Scheme``
   induction for objects in type :n:`@reference`.
 
   :n:`@ident`
-    The name of the scheme. If not provided, the scheme name will be determined automatically
-    from the sorts involved.
+    The name of the scheme. If not provided, the scheme name will be determined
+    automatically from the sorts involved.
+
+  :n:`Induction fo @reference Sort @sort_family`
+    Defines a dependent elimination principle for a family of `@sort_family`s.
 
   :n:`Minimality for @reference Sort @sort_family`
-    Defines a non-dependent elimination principle more natural for inductively defined relations.
+    Defines a non-dependent elimination principle for a constant family of `@sort_family`.
+
+  :n:`Elimination for @reference Sort @sort_family`
+    Defines a dependent case-analysis for a family of `@sort_family`s.
+
+  :n:`Case for @reference Sort @sort_family`
+    Defines a non-dependent case-analysis for a constant family of
+    `@sort_family`.
+
+.. cmd:: Scheme Equality for @reference
 
-  :n:`Equality for @reference`
-     Tries to generate a Boolean equality and a proof of the decidability of the usual equality.
-     If :token:`reference` involves other inductive types, their equality has to be defined first.
+   A high-level tool for which tries to generate a Boolean equality and a proof
+   of the decidability of the usual equality. If :token:`reference` involves
+   other inductive types, their equality has to be defined first.
```

Comment:
The first part should be more consistent with the existing wording for Scheme so users don't have to guess whether different language means different functionality (e.g. "A high-level tool for automatically generating (possibly mutual) induction principles for given types and sorts.v"). -- if accurate

```suggestion
   A high-level tool that tries to generate a Boolean equality and a proof
   of the decidability of the usual equality. If :token:`reference` involves
   other inductive types, their equality has to be defined first.
```

"involves" is vague, do you mean "contains"?